docs: :memo: end of the year summary post (#87)
seedcase-project/community: posts/end-of-year-summary-2024.qmd

---
title: "End of year summary for 2024"
toc-depth: 1
author: Luke W. Johnston
date: 2025-01-23
---

::: callout-note
We need to submit annual reports to Novo Nordisk Foundation as part of
our grant agreement. Since I need to write something anyway, I might as
well make it public and share it with anyone interested in our work over
the years. It's also a good way to reflect on what we've done and any
challenges, some expectations for the upcoming year, and what we've
learned. Plus, it's easier to find when I have to make further reports
over the years.
:::

## Changes and major activities

This past year has been a pretty productive year for us and we're making
a lot more progress now. These are some events or changes that have
happened in 2024.

### New team member!

We hired another research software engineer, Marton Vago, who started in
May, 2024. :tada: He has been a great addition to the team and has

> helped us move forward in many ways with his knowledge and experience!

Reviewer: 🎉

reflecting on our work, and iteratively designing and implementing the
software.

Throughout the year, our regular retrospectives, where we reflect on how
we work, have been steadily been on what works well and less on what to
improve. Which is a good sign that we have worked through many issues
and have a good team flow now.

We've nearly finished our DevOps (development operations) pipelines,
which make it much easier for us to build reliable, robust, tested, and
well-written software that automatically deploys as releases and builds
the documentation as websites. This means we can spend more time and
focus writing well-designed code rather than on the repetitive tasks of
building and deploying software. And any future software we develop, we
will be able to develop and deploy it rapidly, with minimal manual
input.

Through these changes, we've built a strong foundation for working on
our products in this and the upcoming years.

### New collaborations

Some of the more exciting news this year is two new collaborations we
have with newly funded and large research projects. The first is the
[DP-Next](https://dp-next.github.io/) project, which is funded under the

> Steno National Collaborative Grant from NovoNordiskFonden. This project

Reviewer: ```suggestion
Steno National Collaborative Grant from Novo Nordisk Foundation. This project
```
? :)

aims to develop a sustainably effective strategy for prevention of type
2 diabetes. The second is the
[ON-LiMiT](https://steno-aarhus.github.io/ON-LiMiT/) project, which is
funded by the NovoNordiskFonden. It is a large trial to investigate the
effects of lifestyle interventions on remission of diabetes.

While our role in ON-LiMiT is still being defined and discussed, in both

> projects our main tasks will revolve: around establishing their data

Reviewer: ```suggestion
projects our main tasks will revolve around establishing their data
```

Author: Hmm, I put those in because it's a long sentence and couldn't think of how to shorten it. Maybe I'll just try to shorten it again.

> infrastructures as they collect data by using our software; and,

Reviewer: ```suggestion
infrastructures as they collect data by using our software and
```
